kits/accessibility-remediation-copilot/.env.example

```
@@ -0,0 +1,4 @@
+LAMATIC_API_KEY=
+LAMATIC_PROJECT_ID=
+LAMATIC_API_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LAMATIC_API_URL key should go before the LAMATIC_PROJECT_ID key

(UnorderedKey)